bundles/org.openhab.binding.siemensrds/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="siemensrds"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="climatixic">
+
+		<label>Climatix IC</label>
```

Comment:
Shouldn't this be called something like "Account" or similar to make clear what it is about? The bridge type id might also better be adapted.

Reply:
I think "Account" may be too generic; we are referring to an account on the Siemens Climatix IC cloud server ( www.climatixic.com ) so perhaps "Siemens Climatix IC Account" would make it clearest for the user.

Reply:
Sure.`

Reply:
Ok.